docs/api/title.md

```diff
@@ -22,680 +17,53 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
 <EncapsulationPill type="shadow" />
 
-<h2 className="table-of-contents__title">Contents</h2>
-
-<TOCInline
-  toc={toc}
-  maxHeadingLevel={2}
-/>
-
-
 
 `ion-title` is a component that sets the title of the `Toolbar`.
```

Comment:
```suggestion
Title is a text component that sets the title for a [toolbar](./toolbar). It can be used to describe the screen or section a user is currently on or the app being used.
```

Feel free to change the content here, just trying to fluff it a bit.